drivers/spi/spi_context.h

```diff
@@ -185,7 +178,8 @@ static inline void spi_context_cs_configure(struct spi_context *ctx)
 	}
 }
 
-static inline void spi_context_cs_control(struct spi_context *ctx, bool on)
+static inline void _spi_context_cs_control(struct spi_context *ctx,
+					   bool on, bool force_off)
 {
 	if (ctx->config->cs && ctx->config->cs->gpio_dev) {
```

Comment:
could you add (ctx->config) check:
if (ctx->config && ctx->config->cs && ctx->config->cs->gpio_dev)

Reply:
sure, good catch